app/models/Product.scala

```diff
@@ -1,21 +1,22 @@
 package models
 
+import play.api.db.{Database, NamedDatabase}
 import play.api.libs.json.{JsMacroImpl, Json, OFormat}
 
+import javax.inject.Inject
+import scala.concurrent.Future
 import scala.language.experimental.macros
 
 trait ImplicitJsonFormat
```

Comment:
Bold, but remember that this means you are relying on the standard macro to apply to all case classes

Reply:
I will keep that in mind.